Automattic/wp-calypso
# Add domain header notices on domain settings page  🚧

## Changes proposed in this Pull Request

Depends on D70852-code

This PR add domain notices on the header of domain setting page, according to new Figma design (hlVh2q24ad6MCwlwNnE9PQ-fi)

![Markup on 2022-01-07 at 12:20:16](https://user-images.githubusercontent.com/2797601/148536947-88e367e7-8a93-4fbe-b1ae-688004f1a27a.png)

## Testing instructions

To test different cases it could be useful apply this patch on the sandbox (2b3dd-pb/)

- Build this branch locally or open the live Calypso link
- If you're in the live Calypso link, please append `?flags=domains/settings-page-redesign` to your URL to enable the appropriate feature flag
- Go to "Upgrades -> Domains"
- Select different domains and verify that for each case the notice is show according the [Domain messages file](https://docs.google.com/spreadsheets/d/1311HcXoEvPNyMy_FvrzWO0z2nrT5mflPV-_7qdo0Smo/edit#gid=1768180225)
